localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -378,4 +378,32 @@
  underking_male_nerubian_king:0 "Underking"
  underqueen_female_nerubian_king:0 "Underqueen"
  king_male_nerubian_emperor:0 "King"
- queen_female_nerubian_emperor:0 "Queen"
\ No newline at end of file
+ queen_female_nerubian_emperor:0 "Queen"
+
+ legionlord_male:0 "Legionlord"
+ legionlord_female:0 "Legionlord"
+ legion:0 "Legion"
+ 
+ demon_king_male:0 "Demon King"
+ demon_queen_female:0 "Demon Queen"
+ demon_kingdom:0 "Demon Kingdom"
+ 
+ overseer_male:0 "Overseer"
+ overseer_female:0 "Overseer"
+ vile_duchy:0 "Vile Duchy"
+ 
+ lord_demonic_male:0 "Lord"
+ lady_demonic_female:0 "Lady"
+ lordship_demonic:0 "Lordship"
+ 
+ grand_prince_eldritch_male:0 "$grand_prince_nightborne_male$"
+ grand_princess_eldritch_female:0 "$grand_princess_nightborne_female$"
+ grand_principality_eldritch:0 "$grand_principality_nightborne$"
```

Comment:
These references do not refer to an existing title, as 'grand_prince_nightborne_male', 'grand_princess_nightborne_female' &  'grand_principality_nightborne' do not exist. The only references are 37 to 41, but they do not include grand_prince(ss) or even prince(ss). Was the intention perhaps to refer to HIGHborne rather than NIGHTborne?

Reply:
Fixed in 46debb063e34e82bfcfe7e16d63280f61a0c50f1.